File: CAPITULO 01/C++/ex17.c
#include <stdio.h>
int main(){
    float deposito,valor1,valor2,valor_cheque1,valor_cheque2,cpmf,saldo;
    printf("Digite o valor do deposito :");
    scanf("%f%*c",&deposito);
    printf("Digite o valor dos primeiro cheque emitidos: ");
    scanf("%f%*c",&valor1);
    printf("Digite o valor do segundo cheque emitido: ");
    scanf("%f%*c",&valor2);
    cpmf = 0.38;
    valor_cheque1 = valor1 * cpmf / 100;
    valor_cheque2 = valor2 * cpmf / 100;
    saldo = deposito-valor1-valor2-valor_cheque1-valor_cheque2;
    printf("%.2f\n",saldo);

    system("Pause");
    return 0;
}

File: CAPITULO 01/Java/ex17.java
import java.util.*;
import java.io.*;
import java.text.*;
public class ex17{
	public static void main (String[]args){
	
		double deposito,cheque1,cheque2,saldo,cpmf,cpmf1,cpmf2,cpmfTotal;
		Scanner dado = new Scanner (System.in);
		DecimalFormat df = new DecimalFormat("0.00");
		System.out.println("Digite o valor do deposito :");
		deposito = dado.nextDouble();
		System.out.println("Digite o valor do primeiro cheque emitido :");
		cheque1 = dado.nextDouble();
		System.out.println("Digite o valor do segundo cheque emitido :");
		cheque2 = dado.nextDouble();
		cpmf = 0.38;
		cpmf1 = cheque1 * cpmf / 100;
		cpmf2 = cheque2 * cpmf /100;
		cpmfTotal = cpmf1 + cpmf2;
		saldo = deposito - cheque1 - cheque2 - cpmfTotal;
		System.out.println(" "+df.format(saldo));
		
	}
}